recipes-kernel/linux/linux-nilrt_git.bb

```diff
@@ -1,10 +1,10 @@
 DESCRIPTION = "Linux kernel based on nilrt branch"
 NI_RELEASE_VERSION = "22.8"
-LINUX_VERSION = "5.10"
+LINUX_VERSION = "5.15"
```

Comment:
I don't think we should upgrade the kernel to 5.15 in the sumo branch. We have not tested sumo with 5.15 and we're shipping hardknott so we should only put patches in sumo not upgrades.

Reply:
@gratian - Can you create a nilrt/22.8/5.10 kernel branch, then? This was failing to build before I switched to the 5.15 version because there's no 22.8 branch of that kernel. For 22.5, we still had a 5.10 branch so that problem didn't happen in the last release.

Reply:
Yes. Branch nilrt/22.8/5.10 created.

Reply:
Built properly now. Thanks!